contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -938,11 +1062,14 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     function _updateRebalanceState(
         uint256 _chunkRebalanceNotional,
         uint256 _totalRebalanceNotional,
-        uint256 _newLeverageRatio
+        uint256 _newLeverageRatio,
+        string memory _exchangeName
     )
         internal
     {
-        lastTradeTimestamp = block.timestamp;
+        globalLastTradeTimestamp = block.timestamp;
```

Comment:
Could be cleaner if we consolidate these 2 lines into another internal function _updateLastTradeTimestamp function since its used across _updateRebalanceState, _updateIterateState, _updateRipcordState